spec/openstudio/runner_config_spec.rb

```diff
@@ -4,32 +4,52 @@
 # *******************************************************************************
 
 require 'json'
+require 'parallel'
+
 RSpec.describe OpenStudio::Extension::RunnerConfig do
-  before :all do
-    if File.exist? 'runner.conf'
-      puts 'removing runner conf'
-      File.delete('runner.conf')
+  before :each do
+    @test_folder = File.join(File.dirname(__FILE__), "test_defaults")
+    if File.exist?(@test_folder)
+      FileUtils.rm_rf(@test_folder)
```

Comment:
changed the entire spec to use a throwaway folder instead of messing with the root of the repo.